environments/container/README.md

```diff
@@ -0,0 +1,30 @@
+# Virtual Machine environment
+
+This inventory is used to provision a container with the ``*.local.cnx.org`` domains.
+
+## Prerequisite
+
+You should add the following to your host system's ``/etc/hosts`` file when using this environment.
+
+```
+localhost  local.cnx.org archive.local.cnx.org authoring.local.cnx.org legacy.local.cnx.org zope.local.cnx.org
```

Comment:
This didn't work for me. Had to tack these entries onto the `127.0.0.1    localhost` line for some reason.